WaterConsumption/examples/example.jsonld

```diff
@@ -0,0 +1,138 @@
+{
+    "id": "urn:ngsi-ld:Consumer:Consumer139E643",
+    "type": "Consumer",
+    "https://ontology.eglobalmark.com/WaterSmartMeter#acquisitionStageFailure": {
```

Comment:
The names of the keys should not be long iri (they depend on the context but the short ones). In this case, the context is a specific one, not a general one.